cypress/integration/specs/nav.spec.ts

```diff
@@ -0,0 +1,143 @@
+import {NavbarPage} from '../../pages/navbarPage';
+
+describe('Navbar', () => {
+  before(() => {
+    NavbarPage.visitHome();
+  });
+  describe('Navbar Cluster Items', () => {
+    it('cluster', () => {
+      NavbarPage.clickItem('#nav-cluster');
```

Comment:
It would be very good to have IDs like #nav-cluster in the PO (page object) file.

Reply:
Made the required changes.